libraries/Microsoft.Bot.Builder.Expressions.Parser/Expression.g4

```diff
@@ -32,7 +32,7 @@ fragment DIGIT : [0-9];
 
 NUMBER : DIGIT + ( '.' DIGIT +)? ;
 
-WHITESPACE : (' '|'\t') -> skip;
+WHITESPACE : (' '|'\t'|'\u00a0') -> skip;
```

Comment:
let‘s use a fragment here, maybe called UNICODE_SPACE to avoid hardcode all of it